### ISSUE TITLE
Event Definitions do not trigger first message

### ISSUE THREAD

**Issue by leoborges2005** [Graylog2/graylog2-server#8424]
Event Definitions do not trigger first message

## Expected Behavior
Event definitions trigger the first message

## Current Behavior
I used NXLog to send Windows logs to Graylog, and in NXLog configurations I selected only Active Directory group membership changes, filtering by Eventid.
Graylog is receiving logs only from one server, and only AD group membership changes.

I used NXLog to send Windows logs to Graylog and, in the NXLog settings, I selected only the changes from Active Directory group members, filtering by Eventid.

Graylog is receiving logs from only one server and only when members of AD groups are changed.

In Event Definitions, the query was created to select only a few groups, using the TargetUserName field.

When the first member change occurs in an AD group that is part of the Event Definition query, this message is not detected by the Event Definition rule. When the second change occurs, Event Definitions raises the alarm for the first change.

I noticed in the Event Definitions details that the Last Execution field is not filled in until the first message is received, and after the first message is received it starts executing, but only generates alerts from the second message, but alerting about the first , even after hours of difference between the first and the second.

## Possible Solution
I think this is related to the configuration I made so that only relevant events are sent to Graylog, causing very few messages to be sent, causing Event Definition to start its timer only after the first message received by Graylog after its configuration, and in my case, the first message should already be detected by the Event Definition query, but it didn't even start its timer.

## Steps to Reproduce (for bugs)

1. Send logs from only one server and filter to send only relevant messages
2. Create an event definition with the same filter used to send the logs from the message source server.
3. Send the first message to be sent and check if the alarm was generated. It should not have been generated, even in agreement with the consultation.
4. Send the second message to be sent. The first message alert should be generated.
5. For each message generated, the alert for the previous message will be generated.

## Context
As my environment is small, I did not want to send all messages generated on my server to Graylog, for several reasons. Thus, it would be interesting if the alerts were generated from the first message.

## Your Environment

* Graylog Version: v3.3.1+e2d4365
* Elasticsearch Version: 6.8.10
* MongoDB Version: 4.0.19
* Operating System: RedHat 8
* Browser version: Google Chrome  83.0.4103.106

**Comment by leoborges2005** [comment]
![image](https://user-images.githubusercontent.com/67383767/85713552-dd896800-b6bf-11ea-94a0-fe2167cafd15.png)

**Comment by marick7** [comment]
Hey @leoborges2005 did you manage to solve it? I have the same problem  :(

**Comment by patrickmann** [comment]
Duplicate of Graylog2/graylog2-server#6770. Please see the related PR for a full explanation.
Fixed in GL 5.0.
